Source/Core/Core/Config/NetplaySettings.cpp

```diff
@@ -35,4 +35,22 @@ const ConfigInfo<bool> NETPLAY_USE_UPNP{{System::Main, "NetPlay", "UseUPNP"}, fa
 
 const ConfigInfo<bool> NETPLAY_ENABLE_QOS{{System::Main, "NetPlay", "EnableQoS"}, true};
 
+// Host.NetPlay
```

Comment:
This isn't the semantics of these comments, we're still in `Main.NetPlay`. Having the host prefix on all of these options is probably redundant anyways, as it's like there's a non-host variant of them.

Reply:
Indeed I thought about that half-way through it, I don't have strong opinion but it's a bit more clear, do you care more about the prefixes in the code or in the .ini ?

Reply:
Both honestly. I don't really see the need to clarify that these are host-only options, as it's kind of already implied by the fact that the checkboxes aren't visible to non-hosts. Also, buffer size and record inputs aren't host-only options.